theworkoutinfo/ViewControllers/HomeViewController.swift

```diff
@@ -90,4 +133,15 @@ class HomeViewController: UIViewController {
         viewModel.tryLogout.onNext(())
     }
     
+    @objc func showFilterByView() {
+        if tableViewTopSpaceConstraint.constant == 0 {
```

Comment:
These changes in the UI should be called by an event in the viewModel and not from the navigation button directly